2015-12-04 00:17:51,809 ERROR base 3695 -1407198400 Internal Server Error: /shambhala/prevfiles/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 236, in dispatch
    return super(HarmonyPrevFiles, self).dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 154, in get
    context = self.get_context_data(**kwargs)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 243, in get_context_data
    dirs = os.listdir(path)
OSError: [Errno 2] No such file or directory: '/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/oncoFinder2/media/XPN/'
2015-12-04 00:19:18,826 ERROR base 3695 -1423983808 Internal Server Error: /shambhala/prevfiles/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 236, in dispatch
    return super(HarmonyPrevFiles, self).dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 154, in get
    context = self.get_context_data(**kwargs)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 243, in get_context_data
    dirs = os.listdir(path)
OSError: [Errno 2] No such file or directory: '/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/oncoFinder2/media/XPN/'
2015-12-04 00:20:45,796 ERROR base 6118 -1357833408 Internal Server Error: /shambhala/done/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 121, in dispatch
    return super(ShambalaDone, self).dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 154, in get
    context = self.get_context_data(**kwargs)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 127, in get_context_data
    raise
TypeError: exceptions must be old-style classes or derived from BaseException, not NoneType
2015-12-04 00:26:57,367 ERROR base 6204 -1357833408 Internal Server Error: /shambhala/done/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 137, in get_response
    response = response.render()
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 80, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 56, in resolve_template
    return loader.select_template(template)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 180, in select_template
    return get_template(template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 138, in get_template
    template, origin = find_template(template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 127, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 43, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 49, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 149, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/debug_toolbar/panels/templates/panel.py", line 101, in new_template_init
    old_template_init(self, template_string, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 125, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 153, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 215, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 190, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 806, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 276, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 331, in invalid_block_tag
    (command, get_text_list(["'%s'" % p for p in parse_until])))
TemplateSyntaxError: Invalid block tag: 'endif', expected 'empty' or 'endfor'
2015-12-04 00:29:17,338 ERROR base 9508 -1357833408 Internal Server Error: /shambhala/done/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/utils/decorators.py", line 25, in bound_func
    return func(self, *args2, **kwargs2)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 121, in dispatch
    return super(ShambalaDone, self).dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py", line 154, in get
    context = self.get_context_data(**kwargs)
  File "/home/mikhail/EclipseWorkspace/oncoFinder2/oncoFinder2/core/harmony_views.py", line 126, in get_context_data
    context['documents'] =  ShambalaDocument.objects.all().orderby('created_at')
AttributeError: 'QuerySet' object has no attribute 'orderby'
2015-12-04 00:30:58,233 ERROR base 9669 -1367344320 Internal Server Error: /shambhala/done/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 137, in get_response
    response = response.render()
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 80, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 56, in resolve_template
    return loader.select_template(template)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 180, in select_template
    return get_template(template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 138, in get_template
    template, origin = find_template(template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 127, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 43, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 49, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 149, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/debug_toolbar/panels/templates/panel.py", line 101, in new_template_init
    old_template_init(self, template_string, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 125, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 153, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 215, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 190, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 806, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 276, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 331, in invalid_block_tag
    (command, get_text_list(["'%s'" % p for p in parse_until])))
TemplateSyntaxError: Invalid block tag: 'endif', expected 'empty' or 'endfor'
2015-12-04 00:31:01,849 ERROR base 9669 -1386222784 Internal Server Error: /shambhala/done/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 137, in get_response
    response = response.render()
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 80, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/response.py", line 56, in resolve_template
    return loader.select_template(template)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 180, in select_template
    return get_template(template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 138, in get_template
    template, origin = find_template(template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 127, in find_template
    source, display_name = loader(name, dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 43, in __call__
    return self.load_template(template_name, template_dirs)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 49, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader.py", line 149, in get_template_from_string
    return Template(source, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/debug_toolbar/panels/templates/panel.py", line 101, in new_template_init
    old_template_init(self, template_string, origin, name)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 125, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 153, in compile_string
    return parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 215, in do_extends
    nodelist = parser.parse()
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/loader_tags.py", line 190, in do_block
    nodelist = parser.parse(('endblock',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python2.7/dist-packages/django/template/defaulttags.py", line 806, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 276, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/usr/local/lib/python2.7/dist-packages/django/template/base.py", line 331, in invalid_block_tag
    (command, get_text_list(["'%s'" % p for p in parse_until])))
TemplateSyntaxError: Invalid block tag: 'endif', expected 'empty' or 'endfor'
